docs/fundamentals/aspire-sdk-templates.md

```diff
@@ -0,0 +1,165 @@
+---
+title: .NET Aspire templates
+description: Learn how to install the .NET Aspire templates, and how to use them to create new apps.
+ms.date: 11/04/2024
+zone_pivot_groups: dev-environment
+uid: dotnet/aspire/templates
+---
+
+# .NET Aspire templates
+
+There are a number of .NET Aspire project templates available to you. You can use these templates to create full .NET Aspire solutions, or add individual projects to existing .NET Aspire solutions.
+
+The .NET Aspire templates are available in the [📦 Aspire.ProjectTemplates](https://www.nuget.org/packages/Aspire.ProjectTemplates) NuGet package.
+
+## Install the .NET Aspire templates
+
+:::zone pivot="visual-studio"
+
+The .NET Aspire templates are installed automatically when you install Visual Studio 17.9 or later.
+
+:::zone-end
+:::zone pivot="vscode,dotnet-cli"
+
+To install these templates, use the [dotnet new install](/dotnet/core/tools/dotnet-new-install) command, passing in the `Aspire.ProjectTemplates` NuGet identifier.
+
+```dotnetcli
+dotnet new install Aspire.ProjectTemplates
```

Comment:
Might be interesting to mention in here that you may require to run `--force` in the case the user already has the Aspire Workload installed, as otherwise they will get an issue telling them that installing this templates would override the ones from the workload.

Reply:
Added